sdk-springboot/pom.xml

```diff
@@ -29,8 +29,21 @@
 
   <properties>
     <maven.deploy.skip>false</maven.deploy.skip>
+    <springboot.version>2.3.5.RELEASE</springboot.version>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>${springboot.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
```

Comment:
Be careful of this segment, this will override `okhttp` version set in `dapr-sdk`, because spring boot also makes their version declaration in `spring-boot-depencies`, which conficts with dapr's.

Reply:
For this one I am OK since users would need to solve a version conflict too. If this becomes a problem we would know in our integration tests. Thanks for pointing this out. Learned one more thing.